modin/engines/ray/task_wrapper.py

```diff
@@ -11,19 +11,72 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""
+The module with helper mixin for executing functions remotely.
+
+To be used as a piece of building a Ray-based engine.
+"""
+
 import ray
 
 
 @ray.remote
 def deploy_ray_func(func, args):  # pragma: no cover
+    """
+    Wrap `func` to ease calling it remotely.
+
+    Parameters
+    ----------
+    func : callable
+        A local function that we want to call remotely.
+    args : dict
+        Keyword arguments to pass to `func` when calling remotely.
+
+    Returns
+    -------
+    ray.ObjectID
+        Ray identifier of the result being put to Plasma store.
+    """
     return func(**args)
 
 
 class RayTask:
+    """Mixin that provides means of running functions remotely and getting local results."""
+
     @classmethod
     def deploy(cls, func, num_returns, kwargs):
+        """
+        Run local `func` remotely.
+
+        Parameters
+        ----------
+        func : callable
+            A function to call.
+        num_returns : int
+            Amount of return values expected from `func`.
+        kwargs : dict
+            Keyword arguments to pass to remote instance of `func`.
+
+        Returns
+        -------
+        ray.ObjectID
```

Comment:
```suggestion
        ray.ObjectRef or list
```

Currently, we use `deploy` function that always returns list of ray.ObjectRef.